MatrixSDKTests/TestPlans/UnitTests.xctestplan

```diff
@@ -35,6 +35,10 @@
         "MXAsyncTaskQueueUnitTests",
         "MXAuthenticationSessionUnitTests",
         "MXBackgroundTaskUnitTests",
+        "MXBeaconInfoUnitTests\/testJSONDictionaryFail()",
```

Comment:
Afaik we prefer to edit this file manually and add only the class name, instead of every method of it.

Reply:
Weird I added the file at the beginning. I don't know it turns by adding only methods.